dev-packages/browser-integration-tests/scripts/detectFlakyTests.ts

```diff
@@ -128,22 +114,7 @@ function getTestPaths(): string[] {
     cwd: path.join(__dirname, '../'),
   });
 
-  return paths.map(p => path.dirname(p));
-}
-
-function logError(error: unknown) {
-  if (process.env.CI) {
-    console.log('::group::Test failed');
-  } else {
-    console.error(' ⚠️ Test failed:');
-  }
-
-  console.log((error as any).stdout);
-  console.log((error as any).stderr);
-
-  if (process.env.CI) {
-    console.log('::endgroup::');
-  }
+  return paths.map(p => `${path.dirname(p)}/`);
```

Comment:
this was a bit buggy - e.g. if one dirname was `suites/tracing/metrics/web-vitals/test.ts`, it would have made it to `suites/tracing/metrics/web-vitals`, which then in turn also matches e.g. `web-vitals-cls` etc. dirs. Now, it should be stricter in that regard.